### PR TITLE
ESPM formatting changes

### LINKED ISSUE #3179: Error Report on Meter Data Upload
**Is your feature request related to a problem? Please describe.**
We are having lots of problems when we try to upload meter data using the "Upload Portfolio Manager Meter Usage" button. We are specifically trying to upload data from the "data request" format from ENERGY STAR Portfolio Manager. See attached file as an example. When we upload the "monthly usage" tab, we often get the error that the data isn't in the required format "An error occurred while processing the file. Please ensure that your file meets the required specifications.", but we don't have any details on what the required specifications are or where/how we didn't meet them. 
[Uploading Monthly Data.xlsx](https://github.com/SEED-platform/seed/files/8319486/Uploading.Monthly.Data.xlsx)

**Describe the solution you'd like**
ENERGY STAR Portfolio Manager has a similar feature where you can upload data via spreadsheet. When the file doesn't meet the required specs, you get an error report that specifies exactly why the file didn't upload. 

image

**Describe alternatives you've considered**
The meter data upload works better when we use the "Download my portfolio" option from ESPM and then upload this file using the "Upload Portfolio Manager Meter Usage" button; however, we don't have all of the data in this format since participants don't "share their property" with us, they simply respond to the data request. 

**Additional context**

image

### PR DESCRIPTION
**Pull request by anchapin**
#### Any background context you want to provide?
ESPM changed the formatting of the Monthly Usage sheets headers which broke meter uploads.

#### What's this PR do?
Revised the code to work with both the old and new formats from ESPM.

#### How should this be manually tested?
Try uploading the meters from this [spreadsheet](https://docs.google.com/spreadsheets/d/1ANIEpa6T7dldVMAnFrGOBtlZUlNUaZL0/edit?usp=share_link&ouid=111561456156631389864&rtpof=true&sd=true) and see if it works.

#### What are the relevant tickets?
Fixes #3179

### CODE REVIEW

**Comment by RDmitchell** [comment]
@anchapin -- see also #3450

File: seed/data_importer/meters_parser.py [hunk 1]
```diff
@@ -465,7 +465,12 @@ class with the data.
 
         results = []
         for raw_reading in raw_data:
-            start_date = datetime.strptime(raw_reading['Month'], '%b-%y')
+            # December 16, 2022 updated code to match ESPM formatting changes
+            try:
+                start_date = datetime.strptime(raw_reading['Month'], '%b-%y')
+            except KeyError:
+                start_date = datetime.strptime(raw_reading['Month '], '%b-%y')
```
Comment (nllong) [inline seed/data_importer/meters_parser.py:472]: ideally we strip the row headers, but this works for now. I bet they are going to fix this one and we will be back to `Month`.

**Review by nllong — APPROVED** [review]
is there a test file that we can add to our tests with this change.  perhaps one like this

https://github.com/SEED-platform/seed/blob/develop/seed/tests/data/Has_Meters_two_buildings.xlsx

File: seed/models/meters.py [hunk 2]
```diff
@@ -94,6 +94,31 @@ class Meter(models.Model):
         # these values are added based on known usage
         'Fuel Oil #2 Use': ENERGY_TYPE_BY_METER_TYPE[FUEL_OIL_NO_2],
         'Diesel #2 Use': ENERGY_TYPE_BY_METER_TYPE[DIESEL],
+
+        # these mappings are assumed based on ESPM values [new format as of 12-16-2022]
+        'Coal Use (Anthracite) - Monthly': ENERGY_TYPE_BY_METER_TYPE[COAL_ANTHRACITE],
```
Comment (nllong) [inline seed/models/meters.py:99]: cool

File: seed/data_importer/meters_parser.py (hunk 1 above)
Comment (anchapin) [reply seed/data_importer/meters_parser.py:472]: Let me know if you want me to adjust to strip the row headers.

**Comment by anchapin** [comment]
> is there a test file that we can add to our tests with this change. perhaps one like this
> 
> [`develop`/seed/tests/data/Has_Meters_two_buildings.xlsx](https://github.com/SEED-platform/seed/blob/develop/seed/tests/data/Has_Meters_two_buildings.xlsx?rgh-link-date=2022-12-16T20%3A18%3A21Z)

I just added a test file in the seed/tests/data folder called "example-data-request-response-new-format.xlsx". Should I replace it with the old version in the tests?

**Pull request merged**